bedrock/firefox/templates/firefox/nothing-personal/index.html

```diff
@@ -46,80 +48,156 @@ <h1 class="c-page-header-logo-fx">Firefox by Mozilla</h1>
       </div>
     </div>
   </header>
-  <main class="c-page-main-content">
-    <section>
-    <div class="c-browser-windows">
-      <div class="c-layered-browser">
-        {% call browser_border(class='c-browser-window-top-wrapper mzp-l-content mzp-t-content-md mzp-t-content-nospace') %}
-        <div class="c-browser-window-top">
-          <h2>We look forward to not getting to know you</h2>
-        </div>
-        {% endcall %}
-
-        {% call browser_border(class='c-browser-window-top-wrapper c-hidden-browser yellow mzp-l-content mzp-t-content-md mzp-t-content-nospace') %}
-        <div class="c-browser-window-top">
-          <!-- Content is kept so it matches the size of the visible container above -->
-          <h2>We look forward to not getting to know you</h2>
-        </div>
-        {% endcall %}
-
-        {% call browser_border(class='c-browser-window-top-wrapper c-hidden-browser dark-yellow mzp-l-content mzp-t-content-md mzp-t-content-nospace') %}
-        <div class="c-browser-window-top">
-          <!-- Content is kept so it matches the size of the visible container above -->
-          <h2>We look forward to not getting to know you</h2>
+  <div class="c-content-layout">
+    <main class="c-main">
+      <section class="c-things">
+        <div class="c-text-section mzp-l-content mzp-t-content-md mzp-t-content-nospace">
+          <h2 class="c-text-title">Ok, we admit it: we’re fast, reliable <span class="u-text-uppercase">and</span> private.</h2>
```

Comment:
Suggestion: This `span` could be an `em` for semantic emphasis as well as presentation (and make it all-caps and non-italic in CSS)